framework/mesh/mesh.h

```diff
@@ -61,6 +61,16 @@ enum MeshAttributes : int
   UNSTRUCTURED = (1 << 5)
```

Comment:
@wdhawkins @andrsd @zhardy-lanl  : I thought we had a discussion about bitwise operator for mesh enum and that we wanted to get rid of them (but we would keep the bitwise ops for source specifications). I just saw this here by chance. Should be create an issue?

Reply:
Done. See #286.